pymc3/data.py

```diff
@@ -385,3 +386,61 @@ def align_minibatches(batches=None):
                 raise TypeError('{b} is not a Minibatch')
             for rng in Minibatch.RNG[id(b)]:
                 rng.seed()
+
+
+class Data:
+    """Data container class that wraps the theano SharedVariable class
+    and let the model be aware of its inputs and outputs.
+
+    Parameters
+    ----------
+    name : str
+        The name for this variable
+    value
+        A value to associate with this variable
+
```

Comment:
I would also add a note to check out `pm.set_data()` so that users know the intended usage.